views/partials/prs.hbs

```diff
@@ -6,14 +6,14 @@
                     href="https://twitter.com/share"
                     data-size="large"
                     data-text="My progress on hacktoberfest {{ prs.length }} / {{prAmount}}"
-                    data-url="https://hacktoberfestchecker.herokuapp.com/?username={{username}}"
+                    data-url="https://hacktoberfestchecker.jenko.me/?username={{username}}"
                     data-hashtags="hacktoberfest, hacktoberfestchecker">
                     <i class="fab fa-twitter fa-lg"></i>
                     Tweet
                 </a>
             </div>
             <div class="p-2"
-                data-href="https://hacktoberfestchecker.herokuapp.com/?username={{username}}"
+                data-href="https://hacktoberfestchecker.jenko.me/?username={{username}}"
```

Comment:
If you rebase this has been updated to use the dynamic hostname.

Reply:
Alright, I just took a look again at the prs.hbs file from the source, I guess there is no need for this anymore since dynamic hostname is better. I will edit this.